src/main/resources/application-preprod-STAGING-slot.yaml

```diff
@@ -0,0 +1,3 @@
+---
+job.scheduler:
+  autoStart: true
```

Comment:
What do we need this file for? Could it be removed

Reply:
it can't be removed